apidoc/Facebook.yml

```diff
@@ -891,6 +891,46 @@ methods:
     since: "6.1.0"
     platforms: [iphone, ipad]
 
+  - name: logPushNotificationOpen
+    summary: Registers opened push with Facebook campaign. 
+    platforms: [iphone, ipad]
+    since: “6.1.0”
+    properties:
+      - name: value
+        summary: Dictionary of data received in push notifications callback
+        type: Dictionary
+
+  - name: setPushNotificationsDeviceToken
+    summary: dictionary of data received in the push notification
+    platforms: [iphone, ipad]
+    since: “6.1.0”
+    properties:
+      - name: value
+        summary: Device Token received when registering for push notifications
+        type: String
+
+  - name: logPushNotificationOpen
+    summary: Registers opened push with Facebook campaign. 
+    platforms: [android]
```

Comment:
I set the name of the method the same as in FB SDK. I am not sure if it is good, but I have separated code in this case for iOS/Android push notifications